public/components/common/seach/autocomplete_service.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import {
+    EuiFieldSearch,
+    EuiFlexItem,
+    EuiListGroup,
+    EuiListGroupItem,
+    EuiText,
+    EuiSelectableTemplateSitewide,
+    EuiSelectableTemplateSitewideOption,
+    EuiLink,
+    EuiFlexGroup
+  } from '@elastic/eui';
+  import {
+    IQueryBarProps
+  } from './search';
+
+const commands: EuiSelectableTemplateSitewideOption[] =[
+        {label: 'dedup'},
+        {label: 'eval'},
+        {label: 'fields'},
+        {label: 'rename'},
+        {label: 'search'},
+        {label: 'sort'},
+        {label: 'stats'},
+        {label: 'where'},
+        {label: 'head'},
+        {label: 'where'},
+        {label: 'top'}
+    ]
+const getQuerySuggestions = () => {
+    
+    const [searchValue, setSearchValue] = useState('');
+    const [isLoading, setLoading] = useState(false);
+    //const [searchRef, setSearchRef] useState<HTMLInputElement | null>(null);
```

Comment:
Is this commented-out code safe to remove?

Reply:
yeah, i think this was eric's old changes. i'll get remove it

Reply:
This wasn't my code change at all. This is a new file you crated that doesn't exist in any other branches.

Reply:
Oh right my bad. I completely forgot i wrote this (think it was when i was trying out different widgets). I'll remove this file altogether because it's redundant now